frontend/src/components/episode/helpers/sort-shownotes/sort-shownotes.ts

```diff
@@ -0,0 +1,7 @@
+import { Shownote } from 'common/types/types';
+
+const getSortedShownotes = ( shownotes: Shownote[]): Shownote[] => {
+  return [...shownotes].sort((a, b) => a.timestamp - b.timestamp);
```

Comment:
we have the similar functionality in several places

let's reuse it and create a `getSortedItems` helper

Something like

```ts
const getSortedItems = <T>(items: T[], (itemA: T, itemB: T) => number): T[] => {
  // your logic
}
```